src/zcl_abapgit_html_viewer_web.clas.abap

```diff
@@ -71,7 +63,28 @@ CLASS ZCL_ABAPGIT_HTML_VIEWER_WEB IMPLEMENTATION.
 
   METHOD zif_abapgit_html_viewer~show_url.
 
-    RETURN.
+    DATA(lv_path) = cl_http_utility=>if_http_utility~unescape_url( mi_server->request->get_header_field( '~path' ) ).
+
+    DATA(js) = |<script>                                  | &&
+    |  function registerLinks() \{                        | &&
+    |    const links = document.getElementsByTagName("a");| &&
+    |    for (let i = 0; i < links.length; i++) \{        | &&
+    |      if (links[i].href.startsWith("sapevent:")) \{  | &&
+    |        links[i].href = "./" + links[i].href;        | &&
+    |      \}        | &&
+    |    \}          | &&
+    |  \}            | &&
+    |registerLinks();| &&
+    |</script></body>|.
```

Comment:
Why not process text right in abap ? a) faster (text) b) server knows more about the context, in particular, base url.

Also (not directly relevant): sapevent might be in other places as well - forms and on-clicks.

Reply:
I'm just testing stuff, this is a quick hack. I don't want to refactor too much in abapGit as of now